nix/sources.json

```diff
@@ -1,4 +1,16 @@
 {
+    "chain-main": {
```

Comment:
maybe this can use the Nix flake instead?

Reply:
Right now, the integration test shell in still managed with traditional nix, the last time I tried to merge all the stuff into the default flake, it'll build unnecessary stuff when building the binary itself.

Reply:
done, added the integration test.